go.mod

```
@@ -12,7 +12,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-retryablehttp v0.6.6
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.9.0
-	github.com/packethost/packngo v0.26.0
+	github.com/packethost/packngo v0.27.1-0.20220928224154-65e26866e234
```

Once https://github.com/packethost/packngo/pull/348 is merged, `go.sum` must be updated and this line is expected to be:
```suggestion
	github.com/packethost/packngo v0.28.0
```